### PR TITLE
Elevator

### PR DESCRIPTION
**Pull request by yousifj**
Elevator moves from top to bottom or bottom to top. You can choose the starting position in the unity editor and adjust the speed.

### CODE REVIEW

**Comment by BUNN1E5** [comment]
~~Setting the objects parent can have some unintended consequences with scale, did you try using a Kinematic Rigidbody on the elevator and using Rigidbody.MovePosition because that works in the physics engine that should solve any problems with stuttering.~~

File: Assets/Scripts/Elevator.cs [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        }    
+
+    }
+
+
+    private void OnTriggerEnter2D(Collider2D collision)
+    {
+        if (collision.GetComponent<PlatformerController>())
+        {
+            player.transform.SetParent(transform);
```
Comment (BUNN1E5) [inline Assets/Scripts/Elevator.cs:69]: so are you saying that only the player will move on platforms? I mean that's fine I just want to know

Comment (yousifj) [reply Assets/Scripts/Elevator.cs:69]: That's correct and I made the trigger thin. So when you jump, you are no longer effected by the elevator movement.

**Review by Kainkun — APPROVED** [review]
Good work! I changed the Z positions to all be 0 you might have noticed some weird behaviors because of that

**Pull request merged**